lib/gateway/make-resolver.js

```diff
@@ -452,7 +452,7 @@ function makeResolver ({ service, createOperation, transformData, isQuery, isRef
       // Trigger preGatewayExecution hook
       let modifiedQuery
       if (context.preGatewayExecution !== null) {
-        ({ modifiedQuery } = await preGatewayExecutionHandler({ schema, document: operation, context }))
+        ({ modifiedQuery } = await preGatewayExecutionHandler({ schema, document: operation, context, service: { name: service.name } }))
```

Comment:
```suggestion
        ({ modifiedQuery } = await preGatewayExecutionHandler({ schema, document: operation, context, service }))
```

Reply:
I've also adjusted the tests for these changes as well, because the service object contains other fields in addition to `name` in `make-resolver.js` (but not in the `gateway.js` hook call)